scripts/technical-debt-metrics.sh

```diff
@@ -78,7 +78,9 @@ tdc () {
 titlesPathsAndRegexes=(
   "porting notes"                  "*"      "Porting note"
   "backwards compatibility flags"  "*"      "set_option.*backward"
-  "skipAssignedInstances flags"    "*"      "set_option tactic.skipAssignedInstances"
```

Comment:
Could they come back?  I forget if someone implemented to not show the debts that report a total of `0` or not.

Reply:
That feature is not implemented right now.

They could come back in theory, we'd rely on code review to catch this.